README.md

```diff
@@ -49,6 +49,18 @@ this may prompt you for sudo access.
     grunt install
     grunt uninstall
 
+## Building a RPM package
+
+We use Docker to build the RPM package in a clean environment, this also avoids
```

Comment:
It isn't necessary because it uses the [Fedora container to build the RPM](https://github.com/GPII/linux/pull/78/files#diff-35b4a816e0441e6a375cd925af50752cR102) package. I believe we don't need to build a container in this case.